book/forms.rst

```diff
@@ -1021,6 +1021,29 @@ to the ``form()`` or the ``form_start()`` helper:
     a PUT, PATCH or DELETE request. Read the cookbook chapter
     ":doc:`/cookbook/routing/method_parameters`" for more information.
 
+.. note::
+
+  Now that the ``createForm()`` method requires the FQCN, you can not longer pass in ``__construct()`` arguments. To get 
```

Comment:
I agree. I've added a note to prevent people getting confused with Forms as services.

This change was primarily because in previous Symfony version I could pass options such as 

`public function __construct($password_required = false) {}`

into the form constructor which would allow me to alter the option throughout the code base. I can no longer do this in Symfony 2.8+, thus having to pass them as options.

I believe it would be overkill to use the Forms as services in this instance.